modules/gnome-text-editor/gnome-text-editor.nix

```
@@ -0,0 +1,29 @@
+{ config, lib, ... }:
+
+let
+  style = config.lib.stylix.colors {
+    template = ../gedit/template.mustache;
+    extension = "xml";
+  };
+in
+{
+  options.stylix.targets.gnome-text-editor.enable =
+    config.lib.stylix.mkEnableTarget "GNOME Text Editor" true;
+
+  config =
+    lib.mkIf
+      (config.stylix.enable && config.stylix.targets.gnome-text-editor.enable)
+      {
+        nixpkgs.overlays = [
+          (_: prev: {
+            gnome-text-editor = prev.gnome-text-editor.overrideAttrs (oldAttrs: {
+              postFixup =
+                (oldAttrs.postFixup or "")
+                + ''
+                  cp ${style} $out/share/gnome-text-editor/styles/stylix.xml
+                '';
```

To avoid edge cases where `oldAttrs.postFixup` does not end with a newline, we ensure there is one:

```suggestion
              postFixup = ''
                ${oldAttrs.postFixup or ""}
                cp ${style} $out/share/gnome-text-editor/styles/stylix.xml
              '';
```